packages/binding-opcua/tsconfig.json

```diff
@@ -2,7 +2,10 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "outDir": "dist",
-        "rootDir": "src"
+        "rootDir": "src",
+        "skipLibCheck": true,
+        "module": "Node16",
+        "moduleResolution": "Node16"
```

Comment:
I don't think those 2 statements are necessary since the root `tsconfig.json` defines them already like this.